bw-dev

```diff
@@ -23,10 +23,16 @@ trap showerr EXIT
 source .env
 trap - EXIT
 
-if docker compose &> /dev/null ; then
-	DOCKER_COMPOSE="docker compose"
+if which podman > /dev/null ; then
+    CONTAINER_COMMAND="podman"
```

Comment:
From a technical point of view, this is not ideal, since it forces podman use _by the mere fact of having installed it_ (for example, to give it a try, but not configured yet).

This could be avoided by making the decision explicit through an environment variable, e.g.:

```sh
CONTAINER_COMMAND="${BW_DEV_CONTAINER_COMMAND:-docker}"
```

However, I must echo @mouse-reeve's comment:

> having support for this in the shortcuts file means that the project is expected to support podman

This is a decision that every project faces at some point: an "alternative" feature is added to the tooling, that later is expected (and rightfully so) to be maintained as a first-class citizen—which is not always possible because core devs might not have the knowledge. The only way around it is to be _very_ sure when introducing it, that this will not happen...

Do you think, in the face of this, that the wrapper script in the Podman docs could be a solution for  your use case?

Reply:
(I wrote above "a wrapper could be set per-project too". We could document how, if that helps; with direnv and  `PATH_add`.)